lib/documentmanager.js

```diff
@@ -266,7 +266,8 @@
 
         var change = changes.shift();
 
-        this._logger.debug("Applying change: ", JSON.stringify(change, null, "  "));
+// this log line is noisy, only uncommnet if you are debugging change handlers
+//        this._logger.debug("Applying change: ", JSON.stringify(change, null, "  "));
```

Comment:
wait, maybe it wasn't my fault!  anyway, I think it is fine/preferable to remove completely.  thats' my 2 cents.